libs/blocks/aside/aside.js

```diff
@@ -14,74 +14,78 @@
 * Aside - v5.1
 */
 
-import { decorateBlockBg, decorateButtons } from '../../utils/decorate.js';
+import { decorateBlockBg, decorateBlockText } from '../../utils/decorate.js';
 import { createTag } from '../../utils/utils.js';
 
-const asideTypes = ['inline', 'notification'];
-const [INLINE, NOTIFICATION] = asideTypes;
-const asideSizes = ['extra-small', 'small', 'medium', 'large'];
-const [SIZE_XS, SIZE_S, SIZE_M, SIZE_L] = asideSizes;
+// standard/default aside uses same text sizes as the split
+const variants = ['split', 'inline', 'notification'];
+const sizes = ['extra-small', 'small', 'medium', 'large'];
+const [split, inline, notification] = variants;
+const [xsmall, small, medium, large] = sizes;
+const blockConfig = {
+  [split]: ['XL', 'S', 'M'],
+  [inline]: ['S', 'M'],
+  [notification]: {
+    [xsmall]: ['M', 'M'],
+    [small]: ['M', 'M'],
+    [medium]: ['S', 'S'],
+    [large]: ['L', 'M'],
+  },
+};
+
+function getBlockData(el) {
+  const variant = variants.find((variantClass) => el.classList.contains(variantClass));
+  const size = sizes.find((sizeClass) => el.classList.contains(sizeClass));
+  const blockData = variant ? blockConfig[variant] : blockConfig[Object.keys(blockConfig)[0]];
+  return variant && size && !Array.isArray(blockData) ? blockData[size] : blockData;
+}
+
+function decorateInlineVideo(el) {
+  const video = el.querySelector('a[href*=".mp4"]');
+  if (!video) return;
+  const parent = video.parentElement;
+  parent.classList.add('milo-inline-video');
+  const source = createTag('source', { src: video.href, type: 'video/mp4' });
+  const html = createTag('video', { playsinline: '', autoplay: '', muted: '', loop: '' }, source);
+  parent.appendChild(html);
+  video.remove();
+}
```

Comment:
@saugatmalla I'm assuming this won't be needed with [your universal approach](https://github.com/adobecom/milo/pull/478), right? What do you think? 
Can you guys sync up to make sure @Sartxi gets what he needs and we don't have duplicate code?

Nit for future reference: It'd be helpful to see a video in your sample page.


 
cc: @chrischrischris

Reply:
> @saugatmalla I'm assuming this won't be needed with [your universal approach](https://github.com/adobecom/milo/pull/478), right? What do you think?

@rgclayton Yes, it won't be needed for the slack uploaded mp4s.

Reply:
@rgclayton okay I will remove this function. Just as a note the Marquee also uses this method

Reply:
@Sartxi Is this supposed to be for background videos?

My understanding of maquee's is that .mp4 only worked as a background video.

Reply:
>@rgclayton Yes, it won't be needed for the slack uploaded mp4s.

I think we need to come to a decisions for .mp4s then to avoid confusion. We shouldn't have to have multiple ways to add .mp4 videos to content. 

cc: @Sartxi

Reply:
Some background: The request for the marquee was an inline looped muted video ("animated marquee"), like you see on the [holiday shopping report](https://business.adobe.com/resources/holiday-shopping-report.html). At the time, we didn't have other support for adding those attributes to the video (not sure if there's something with params now).

Reply:
Agreed. I believe the [auto decorating of mp4](https://github.com/adobecom/milo/pull/478) can now support mp4s in any block, so we can remove other ways to add .mp4.

Reply:
I removed the video decorating because that will be included in the other PR from @saugatmalla